pkg/provider/cloud/kubevirt/vm_instancetype.go

```diff
@@ -65,3 +71,54 @@ func GetKubermaticStandardInstancetypes(client ctrlruntimeclient.Client, getter
 	}
 	return instancetypes
 }
+
+// DescribeInstanceType returns the NodeCapacity from the VirtualMachine instancetype.
+func DescribeInstanceType(ctx context.Context, kubeconfig string, it *kubevirtv1.InstancetypeMatcher) (*provider.NodeCapacity, error) {
+	client, err := NewClient(kubeconfig, ClientOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	switch it.Kind {
+	case "VirtualMachineInstancetype": // "standard" instancetype
+		standardInstancetypes := GetKubermaticStandardInstancetypes(client, &kvmanifests.StandardInstancetypeGetter{})
+		for _, instancetype := range standardInstancetypes {
+			if strings.EqualFold(instancetype.Name, it.Name) {
+				return instanceTypeToNodeCapacity(instancetype.Spec)
+			}
+		}
+
+	default: //  "VirtualMachineClusterInstancetype" and default case = "custom" instancetype.
```

Comment:
I aligned with the KubeVirt type:

> // Kind specifies which instancetype resource is referenced.
> 	// Allowed values are: "VirtualMachineInstancetype" and "VirtualMachineClusterInstancetype".
> 	// If not specified, "VirtualMachineClusterInstancetype" is used by default.
> 	//
> 	// +optional

And went for the same behaviour, as we re-use directly the type.

But you're right, I will go to prevent the default case and raise an error if not specifically specified.